### ISSUE TITLE
Selecting Multiple Entities Sometimes Causes Weird Texture Issues

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#1409]
Doesn't happen everytime, but if you select multiple entities sometimes the textures will bug out. Causes entities to completely change material, sometimes causes them to become transparent. Once this bug kicks in, it will continue to happen until you restart Max. Reloading the level or opening a new level does not fix the bug from happening. The bugged entities will also lose the yellow selection outline even when they are selected. 

I cannot reproduce on a fresh level, only happens to me on levels with a bunch of entities.  I think using the Undo tool will cause it to happen more frequently. 

Video attached. 
https://youtu.be/3ZZBo-WpdsY

The first part of the video happened to me in real time. I wanted to raise that structure up higher and just happened to catch the bug when I shadowplay on. I put the entire clip in there in case the stuff I do right before the bug happens helps solve what caused it... but 45 seconds in is about where the bug occurs. Then as youll see, I was able to reproduce it consistently until I restarted Max. 

Same bug happens with Undo, but is rarer.

**Comment by plemsoft** [comment]
@Kitakazi Thanks for reporting this issue, i was not able to reproduce it , but as you say it only happens sometimes, could it be a memory issue ? , when it happens could you sent me a copy of you "Guru-MapEditor.log" so im able to check it, also let me know how many gb. of memory you have, Thanks :)

**Comment by Kitakazi** [comment]
I have 16gb of memory. 
Bug just happened here is the Guru-MapEdit log
[Guru-MapEditor.log](https://github.com/TheGameCreators/GameGuruRepo/files/7230730/Guru-MapEditor.log)
.

**Comment by Kitakazi** [comment]
This latest update seems that this bug is less common, although it just happened to me, so i'm posting another one of my editorlogs in case it helps. Im not sure it has anything to do with multiple selection anymore, but may have to do with the current objects and groups feature. I've noticed before this bug happens sometimes the "current object" panel will start to display entities that I selected a previously sometimes ones I selected 2+minutes ago, it will display these for a very short time maybe half a second, and this doesn't always happen.  Not sure if that really has anything to do with the texture issues. 

Also I believe it is only the surface map that bugs out. 
[Guru-MapEditor.log](https://github.com/TheGameCreators/GameGuruRepo/files/7275074/Guru-MapEditor.log)
